tutorials/experiment_design_01.ipynb

```diff
@@ -201,7 +202,7 @@
     "t_start = pd.Timedelta(\"0s\")\n",
     "t_end = pd.Timedelta(str(t_weld.to_base_units()))\n",
     "\n",
-    "coords = [tcp_start_point.magnitude, tcp_end_point.magnitude]\n",
+    "coords = np.vstack(tcp_start_point, tcp_end_point)\n",
```

Comment:
using `np.vstack` or `np.append`, `np.hstack` should be our preferred option since it is safe against conversion errors